src/o365/spo/commands/theme/theme-list.ts

```diff
@@ -0,0 +1,118 @@
+import auth from '../../SpoAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import * as request from 'request-promise-native';
+import SpoCommand from '../../SpoCommand';
+import Utils from '../../../../Utils';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: GlobalOptions;
+}
+
+class ThemeListCommand extends SpoCommand {
+
+  public get name(): string {
+    return commands.THEME_LIST;
+  }
+
+  public get description(): string {
+    return 'Retreives the list of custom themes added to the tenant store.';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    auth
+      .ensureAccessToken(auth.service.resource, cmd, this.debug)
+      .then((accessToken: string): request.RequestPromise => {
+
+        if (this.debug) {
+          cmd.log(`Retrieved access token ${accessToken}. Loading themes from tenant store...`);
+        }
+
+        if (this.verbose) {
+          cmd.log(`Retrieving themes from tenant store...`);
+        }
+
+        const requestOptions: any = {
+          url: `${auth.site.url}/_api/thememanager/GetTenantThemingOptions`,
+          method: 'POST',
+          headers: Utils.getRequestHeaders({
+            authorization: `Bearer ${accessToken}`,
+            'accept': 'application/json;odata=nometadata'
+          }),
+          body: '',
+          json: true
+        };
+
+        if (this.debug) {
+          cmd.log('Executing web request...');
+          cmd.log(requestOptions);
+          cmd.log('');
+        }
+        return request.get(requestOptions);
```

Comment:
If this is supposed to be a POST request, we should use `request.post` here instead of `request.get`.

Reply:
agreed. will make the changes!